src/test/java/org/opentripplanner/routing/algorithm/mapping/BikeRentalSnapshotTest.java

```diff
@@ -109,8 +110,11 @@ public void directBikeRental() {
 
         try {
             expectArriveByToMatchDepartAtAndSnapshot(request);
-        } catch (CompletionException e) {
-            RoutingValidationException.unwrapAndRethrowCompletionException(e);
+            throw new IllegalArgumentException();
+        } catch (Exception e) {
+            // for some strange reason the stack trace actually is not printed to the console
+            e.printStackTrace();
+            Arrays.stream(e.getSuppressed()).forEach(Throwable::printStackTrace);
```

Comment:
This is unrelated?

Reply:
This is from https://github.com/opentripplanner/OpenTripPlanner/pull/3894. I was trying to confirm that the failing test actually is caused by a race condition in the turn restrictions code. No luck so far.